Tests/unit/PolarisAuthBasic.Tests.ps1

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+Describe "Polaris Authentication" {
+    
+    BeforeAll {
+        Start-Job -Scriptblock {
+            Import-Module $using:PSScriptRoot\..\..\Polaris.psd1
+
+            New-PolarisRoute -Path /helloworld -Method GET -Scriptblock {
+                $Response.Send('Hello World')
+            }
+
+            $Polaris = Start-Polaris -MinRunspaces 1 -MaxRunspaces 5 -Auth Basic
```

Comment:
The `-MinRunspaces 1 -MaxRunspaces 5` was a back-compat thing. You shouldn't need that and it actually doesn't do anything :smile:

Reply:
Removed!